components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -876,25 +863,25 @@ bool AdsServiceImpl::MigratePrefs(
   return true;
 }
 
-void AdsServiceImpl::MigratePrefsVersion1To2() const {
-  DCHECK_EQ(1, GetPrefsVersion()) << "Invalid migration path";
```

Comment:
why this was removed?

Reply:
Because during a dry run the prefs version is not updated, therefore with the fix the DCHECK failed